ci/build-lint-preview.sh

```diff
@@ -61,3 +69,16 @@ docker run \
         "/var/logs/errors.txt"
 
 display_annotations
+
+echo "--- Uploading docs :outbox_tray:"
+improbadoc upload \
+    "${DOCS_TYPE}" \
+    "${DOCS_PATH_TMP}" \
```

Comment:
do we need to clean the temp folder?

Reply:
nope, it'll get cleaned by the global pre-exit hook
![image](https://user-images.githubusercontent.com/10051819/62145852-11abab80-b2ec-11e9-918d-b9d60cc3e03e.png)